lib/verbalize.rb

```diff
@@ -79,13 +88,16 @@ def __verbalized_send(method_name, *args)
       end
 
       Failure.new(error)
+    rescue VerbalizeError => e
+      Failure.new(e.failure)
```

Comment:
on second thought this doesn't make sense, as it would require tests stubbing failures to raise an instance of `Verbalize::VerbalizeError` with an instance of `Failure` instead of just the failure object.